<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url('https://themes.googleusercontent.com/fonts/css?kit=W8Jl-jxzzEnyRbrIe94pac0zaykDSyyRzuugVKVeSl4');ol.lst-kix_hz3552dmzdc3-1.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-1 0}ol.lst-kix_hz3552dmzdc3-6{list-style-type:none}ol.lst-kix_hz3552dmzdc3-5{list-style-type:none}.lst-kix_list_2-6>li:before{content:"\0025a0  "}.lst-kix_list_2-7>li:before{content:"\0025a0  "}ol.lst-kix_hz3552dmzdc3-4{list-style-type:none}ol.lst-kix_hz3552dmzdc3-3{list-style-type:none}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_list_2-4>li:before{content:"\0025a0  "}.lst-kix_list_2-5>li:before{content:"\0025a0  "}.lst-kix_list_2-8>li:before{content:"\0025a0  "}ol.lst-kix_hz3552dmzdc3-8{list-style-type:none}ol.lst-kix_hz3552dmzdc3-7{list-style-type:none}ol.lst-kix_hz3552dmzdc3-8.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-8 0}.lst-kix_hz3552dmzdc3-0>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-0}.lst-kix_hz3552dmzdc3-5>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-5}.lst-kix_hz3552dmzdc3-6>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-6}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ol.lst-kix_hz3552dmzdc3-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ol.lst-kix_hz3552dmzdc3-1{list-style-type:none}ol.lst-kix_hz3552dmzdc3-5.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-5 0}ul.lst-kix_list_1-8{list-style-type:none}ol.lst-kix_hz3552dmzdc3-0{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}.lst-kix_hz3552dmzdc3-7>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-7}.lst-kix_hz3552dmzdc3-8>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-8,lower-roman) ". "}.lst-kix_hz3552dmzdc3-5>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-5,lower-roman) ". "}ol.lst-kix_hz3552dmzdc3-2.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-2 0}.lst-kix_hz3552dmzdc3-7>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-7,lower-latin) ". "}.lst-kix_hz3552dmzdc3-6>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-6,decimal) ". "}.lst-kix_hz3552dmzdc3-1>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-1}.lst-kix_hz3552dmzdc3-4>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-4}ol.lst-kix_hz3552dmzdc3-6.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-6 0}ol.lst-kix_hz3552dmzdc3-3.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-3 0}.lst-kix_hz3552dmzdc3-3>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-3}.lst-kix_hz3552dmzdc3-8>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-8}.lst-kix_hz3552dmzdc3-1>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-1,lower-latin) ". "}ul.lst-kix_list_2-8{list-style-type:none}.lst-kix_hz3552dmzdc3-0>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-0,decimal) ". "}.lst-kix_hz3552dmzdc3-4>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-4,lower-latin) ". "}.lst-kix_hz3552dmzdc3-2>li{counter-increment:lst-ctn-kix_hz3552dmzdc3-2}ul.lst-kix_list_2-2{list-style-type:none}.lst-kix_list_1-0>li:before{content:"\0025cf  "}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}.lst-kix_hz3552dmzdc3-3>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-3,decimal) ". "}ul.lst-kix_list_2-6{list-style-type:none}ol.lst-kix_hz3552dmzdc3-0.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-0 0}.lst-kix_list_1-1>li:before{content:"\0025cb  "}.lst-kix_list_1-2>li:before{content:"\0025a0  "}ul.lst-kix_list_2-7{list-style-type:none}.lst-kix_hz3552dmzdc3-2>li:before{content:"" counter(lst-ctn-kix_hz3552dmzdc3-2,lower-roman) ". "}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}.lst-kix_list_1-3>li:before{content:"\0025a0  "}.lst-kix_list_1-4>li:before{content:"\0025a0  "}.lst-kix_list_1-7>li:before{content:"\0025a0  "}.lst-kix_list_1-5>li:before{content:"\0025a0  "}.lst-kix_list_1-6>li:before{content:"\0025a0  "}ol.lst-kix_hz3552dmzdc3-7.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-7 0}ol.lst-kix_hz3552dmzdc3-4.start{counter-reset:lst-ctn-kix_hz3552dmzdc3-4 0}.lst-kix_list_2-0>li:before{content:"\0025cf  "}.lst-kix_list_2-1>li:before{content:"\0025cb  "}.lst-kix_list_1-8>li:before{content:"\0025a0  "}.lst-kix_list_2-2>li:before{content:"\0025a0  "}.lst-kix_list_2-3>li:before{content:"\0025a0  "}ol{margin:0;padding:0}table td,table th{padding:0}.c12{border-right-style:solid;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:234pt;border-top-color:#000000;border-bottom-style:solid}.c4{border-right-style:solid;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:117pt;border-top-color:#000000;border-bottom-style:solid}.c2{border-right-style:solid;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:156pt;border-top-color:#000000;border-bottom-style:solid}.c15{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:24pt;font-family:"Arial";font-style:normal}.c26{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Courier";font-style:normal}.c5{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c16{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c11{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:24pt;font-family:"Courier";font-style:normal}.c14{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:italic}.c23{color:#000000;text-decoration:none;vertical-align:baseline;font-size:18pt;font-family:"Arial";font-style:normal}.c10{font-weight:400;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c6{padding-top:12pt;padding-bottom:12pt;line-height:1.0;text-align:left}.c22{padding-top:11.2pt;padding-bottom:11.2pt;line-height:1.0;text-align:left}.c7{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:right}.c3{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#0000ee;text-decoration:underline}.c13{border-spacing:0;border-collapse:collapse;margin-right:auto}.c1{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c24{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c17{margin-left:36pt;padding-left:0pt}.c25{padding:0;margin:0}.c8{color:inherit;text-decoration:inherit}.c9{height:11pt}.c0{height:0pt}.c19{font-style:italic}.c21{height:18pt}.c18{font-weight:700}.c20{height:24pt}.title{padding-top:24pt;color:#000000;font-weight:700;font-size:36pt;padding-bottom:6pt;font-family:"Arial";line-height:1.0;page-break-after:avoid;text-align:left}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:"Georgia";line-height:1.0;page-break-after:avoid;font-style:italic;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:12pt;color:#000000;font-weight:700;font-size:24pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h2{padding-top:11.2pt;color:#000000;font-weight:700;font-size:18pt;padding-bottom:11.2pt;font-family:"Arial";line-height:1.0;text-align:left}h3{padding-top:12pt;color:#000000;font-weight:700;font-size:14pt;padding-bottom:12pt;font-family:"Arial";line-height:1.0;text-align:left}h4{padding-top:12.8pt;color:#000000;font-weight:700;font-size:12pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h5{padding-top:12.8pt;color:#000000;font-weight:700;font-size:9pt;padding-bottom:12.8pt;font-family:"Arial";line-height:1.0;text-align:left}h6{padding-top:18pt;color:#000000;font-weight:700;font-size:8pt;padding-bottom:18pt;font-family:"Arial";line-height:1.0;text-align:left}</style></head><body class="c24"><h1 class="c6"><span class="c15">Analysis of sheep astrocytes in response to inflammation and cholinergic manipulation</span></h1><p class="c1"><span class="c18">Author</span><span class="c5">: James W. MacDonald.</span></p><p class="c1"><span class="c18">This version edited for GitHub by:</span><span class="c5">&nbsp;Martin G. Frasch.</span></p><p class="c1"><span class="c18">This is the companion repo for arXiv:1812.06617. You can cite it by using the DOI 10.5281/zenodo.2609202.</span>
  <p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c18">Compiled</span><span class="c5">: Fri Mar 22 10:40:22 2019</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c18">Samples used in this analysis.</span><span class="c5">&nbsp;Treat.Number in this table lists the number of treatments the animal received (the single or double hit animals).</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">Astrocytes samples used in this analysis.</span></p><p class="c1 c9"><span class="c5"></span></p><a id="t.1c3ba78b6c3e4d18b268b553c8fb04115e108983"></a><a id="t.0"></a><table class="c13"><tbody><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Sample</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Treatment</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Gender</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Treat.Number</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T1</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T1</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T1</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T1</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T2</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T2</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T2</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T2</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">F</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4502P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4502P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4502P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">4502P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">1</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">711P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">2</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">711P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">2</span></p></td></tr><tr class="c0"><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">711P</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c1"><span class="c5">M</span></p></td><td class="c4" colspan="1" rowspan="1"><p class="c7"><span class="c5">2</span></p></td></tr></tbody></table><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">Microglia samples used in this analysis.</span></p><p class="c1 c9"><span class="c5"></span></p><a id="t.dd43c37ad2a1307c64c437ed9bdd3a051ef4160c"></a><a id="t.1"></a><table class="c13"><tbody><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">File</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Animal</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Treatment</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_1</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">5458T</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">5458T</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_3</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">5458T</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_4</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">5458T</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_5</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">3872T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_6</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">3872T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_7</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">3872T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_8</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">3872T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_9</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Ctrl</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_10</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_11</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+B100</span></p></td></tr><tr class="c0"><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">Mingju_12</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">4414T-R2</span></p></td><td class="c2" colspan="1" rowspan="1"><p class="c1"><span class="c5">LPS100+A10</span></p></td></tr></tbody></table><h1 class="c1 c20"><span class="c15"></span></h1><p class="c1"><span class="c3"><a class="c8" href="pca.html">3D PCA plot</a></span><span>&nbsp;(astrocytes)</span><span class="c5">. PCA, or principal components analysis, is a method to take high dimensional data and reduce to only a few dimensions in order to visualize how similar samples of the same type are. The first dimension captures the largest amount of variability between samples, the second dimension captures the next largest amount of variability, and so on. The above link will load an HTML page with an interactive plot that you can adjust by clicking on the plot and moving your mouse. </span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">These plots indicate that there are large differences between the different treatment groups, and much smaller differences within each group, indicating that we are getting good signal and will likely have lots of differentially expressed genes.</span></p><p class="c6"><span class="c16">Model fit and results</span></p><p class="c1"><span class="c5">To make these comparisons we use the Bioconductor limma package (Smyth 2004), which includes a function that is intended to convert the count data to log counts/million, as well as estimating observation-level weights (e.g. a weight for each gene, for each sample). We can then fit a conventional weighted linear model to these data, as the weights account for any dependence between the mean and variance. For the microglia samples we also compute sample-specific weights that are intended to down-weight any samples that are not very similar to other samples of the same type. This is an alternative to simply excluding the sample.</span></p><p class="c1"><span>We fit the model described above, including a blocking effect for each animal, and made the requested comparisons, incorporating a fold change criterion into the test. In other words, the conventional test for a difference is between the null hypothesis \(H_0: \beta = 0\) versus the alternative hypothesis \(H_A: \beta \neq 0\), but this may include very small changes that are likely to be biologically meaningless. One alternative to exclude such genes is to use a </span><span class="c19">post hoc</span><span class="c5">&nbsp;fold change adjustment, where we select genes based on the observed fold change between groups. This is problematic because we ignore the imprecision in our estimate of fold change. A better method is to incorporate the fold change into our inference, where we test \(H_0: \beta &lt; |c|\) against the alternative \(H_A: \beta \geq |c|\), for some constant fold change. By doing this, we are testing to see if we have evidence that the underlying population differences are larger than a given fold change, rather than simply testing that our sample data fulfill those criteria. We used a 1.5-fold change, and a false discovery rate (FDR) of 0.05 as criteria to define significantly differentially expressed genes.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">We present the number of genes for each comparison, as well as links to results in the table below.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span>Number of significant genes in </span><span class="c18">astrocytes samples</span><span class="c5">&nbsp;at an FDR &lt; 0.05 and a 1.5-fold change.</span></p><p class="c1 c9"><span class="c5"></span></p><a id="t.0fd302cbf957cabe047dc136fec99e3fb182e4e7"></a><a id="t.2"></a><table class="c13"><tbody><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c5">Comparison</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c5">Number significant</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100_1_hit_vs_Ctrl_1_hit.html">LPS100_1_hit vs Ctrl_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">1835</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BA10_1_hit_vs_Ctrl_1_hit.html">LPS100+A10_1_hit vs Ctrl_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">1725</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BB100_1_hit_vs_Ctrl_1_hit.html">LPS100+B100_1_hit vs Ctrl_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">1744</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BA10_1_hit_vs_LPS100_1_hit.html">LPS100+A10_1_hit vs LPS100_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">273</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BB100_1_hit_vs_LPS100_1_hit.html">LPS100+B100_1_hit vs LPS100_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">0</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BA10_1_hit_vs_LPS100%2BB100_1_hit.html">LPS100+A10_1_hit vs LPS100+B100_1_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">292</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100_1_hit_vs_LPS100_2_hit.html">LPS100_1_hit vs LPS100_2_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">3761</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BA10_1_hit_vs_LPS100%2BA10_2_hit.html">LPS100+A10_1_hit vs LPS100+A10_2_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">3307</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports/LPS100%2BB100_1_hit_vs_LPS100%2BB100_2_hit.html">LPS100+B100_1_hit vs LPS100+B100_2_hit</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">3860</span></p></td></tr></tbody></table><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span>Number of significant genes in </span><span class="c18">microglia samples</span><span class="c5">&nbsp;at an FDR &lt; 0.05 and a 1.5-fold change.</span></p><p class="c1 c9"><span class="c5"></span></p><a id="t.d5588f651c15330304ec64dccdfdbb4613b9d16c"></a><a id="t.3"></a><table class="c13"><tbody><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c5">Comparison</span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c5">Number significant</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100_vs_Ctrl.html">LPS100 vs Ctrl</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">1779</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100%2BA10_vs_Ctrl.html">LPS100+A10 vs Ctrl</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">4721</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100%2BB100_vs_Ctrl.html">LPS100+B100 vs Ctrl</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">4049</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100%2BA10_vs_LPS100.html">LPS100+A10 vs LPS100</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">8</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100%2BB100_vs_LPS100.html">LPS100+B100 vs LPS100</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">0</span></p></td></tr><tr class="c0"><td class="c12" colspan="1" rowspan="1"><p class="c1"><span class="c3"><a class="c8" href="reports_micro/LPS100%2BA10_vs_LPS100%2BB100.html">LPS100+A10 vs LPS100+B100</a></span></p></td><td class="c12" colspan="1" rowspan="1"><p class="c7"><span class="c5">1132</span></p></td></tr></tbody></table><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">The links in the above table will load HTML tables containing the genes that were significantly different in this analysis. By default the table shows only 10 rows, and is sorted on the first column. You can change the number of rows using the drop-down at the top left of the page, and can sort on any column by simply clicking on the header of the column you want to sort by. Each row has a small plot showing the relative differences between the two groups, but in order to fit the plot into the table we have to strip off the axis labels. If you right-click on any plot and choose &lsquo;Open in new tab&rsquo;, you will be presented with a full size plot that does have axis labels.</span></p><p class="c1"><span class="c5">The first four columns in the table contain annotation information, including links to Ensembl and NCBI, and the gene symbol and description (if available). We get the symbol and description from Ensembl&rsquo;s Biomart server, and for some reason not all of the symbols and/or descriptions are available from that database. The logFC column shows the log fold change between the groups being compared. The sign of the fold change indicates if the gene is up or down-regulated, and the title of the table indicates the direction of comparison. For example, LPS100_1_hit vs Ctrl_1_hit indicates that we are comparing the LPS treated - Control for the single hit samples, so a positive fold change means the gene is up-regulated in LPS treated samples, and a negative fold change indicates it is down-regulated in LPS treated samples. You can convert the log fold changes to linear scale by exponentiating; if the logFC is 1.23, then the linear fold change is \(2^{1.23}\) = 2.35.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">The logCPM column can be interpreted as a measure of the relative expression for a given gene. All things equal, a larger value indicates a higher level of gene expression. The FDR is a is a measure that estimates the maximum proportion of false discoveries in a given set of genes. We selected genes at an FDR &lt; 0.05, meaning that we expect at most that 5% of the genes selected are false positives.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">We also output the same set of tables in Excel format - you can find them in the spreadsheets subdirectory that accompanies this document. In addition, we output all (post filtered) genes in two Excel spreadsheets that is primarily intended for analysis using IPA (Ingenuity Pathway Analysis), called Frasch_pid1708_astroglia_ingenuity.xlsx and Frasch_pid1708_microglia_ingenuity.xlsx. These files can be useful to look at results for a given gene, across all comparisons, or to simply look for genes that might not appear in the sets of differentially expressed genes.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c22"><span class="c16">Venn diagrams</span></p><p class="c1"><span>There are several sets of comparisons with consistent denominators, such as the three comparisons of treated samples to control samples. It might be of interest to know which genes are unique to a given comparison, and which are shared between two or more of those comparisons. We generated </span><span class="c3"><a class="c8" href="venns/venn.html">Venn diagrams</a></span><span class="c5">&nbsp;for three sets of comparisons.</span></p><p class="c1"><span class="c5">We made a Venn diagram for the three comparisons of treated vs control, a two-way Venn diagram of the agonist + LPS versus either LPS alone or LPS plus antagonist (this shows that there is very little difference between LPS and LPS plus antagonist), and finally we make a Venn diagram of the one-hit versus two-hit for all three treatments.</span></p><p class="c1"><span>Please note that the genes in any intersection between comparisons have to be significantly differentially expressed in both (or all three, depending on the intersection) of the comparisons. In addition, the direction of change has to be the same as well. For example, if a given gene is differentially expressed in LPS vs control and LPS+A10 vs control, </span><span class="c19">and it is either up or down-regulated in both comparisons as well</span><span class="c5">, then it will be listed in the intersection between those two comparisons. If it is significant in both comparisons, but is up-regulated in one comparison, but down-regulated in the other, then it will be listed in the unique portion of the Venn diagram for each comparison.</span></p><p class="c1"><span class="c5">The numbers in each cell of the Venn diagrams are links that will open HTML pages containing the genes in a given cell. The &lsquo;venns&rsquo; subdirectory that accompanies this report also contains text versions of the data in each cell of each Venn diagram.</span></p><h2 class="c1 c21"><span class="c18 c23"></span></h2><p class="c22"><span class="c16">Comparison to microglia samples</span></p><p class="c1"><span class="c14">Please note that differences between the astrocyte and microglia samples are confounded by technical differences. In other words, we cannot say with certainty if any differences between the two cell types represent real biological differences or are simply technical differences due to running samples at slightly different times, or some unknown combination of the two.</span></p><p class="c1 c9"><span class="c14"></span></p><p class="c1"><span class="c5">With this in mind, one additional goal for this analysis was to compare to the previous microglia samples. To do so, we downloaded the FASTQ data from SRA and processed using the same salmon/tximport pipeline we used for the astrocyte samples. The only difference was in the modeling step, where we used the GLM model instead of the quasi-likelihood model. The quasi-likelihood model is more conservative than the GLM (and arguably suffers from fewer false positives). With the microglia data we get no differentially expressed genes when using the quasi-likelihood, so we had to use the GLM framework. We still incorporated the 1.5-fold change into the comparisons and selected genes based on an FDR &lt; 0.05, so the only difference is how we model the gene dispersion.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">First, we can take a look at a PCA plot that includes just the single-hit LPS and control samples for microglia and astrocytes (we exclude the two-hit samples because there is no two-hit control for astrocytes).</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">PCA plot of microglia and astrocyte samples. Here we can see that astrocytes and microglia separate on the first principal component, and the second principal component captures the LPS treatment differences. The third principal component captures some intra-treatment variability for the microglia samples, particularly for one of the LPS treated microglia samples.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span>From the PCA plot it is obvious that the main differences between samples are captured on the first two principal components, so a 2D plot may be more useful. Regardless, a high resolution tiff can be found </span><span class="c3"><a class="c8" href="pdfs/PCA.3D_microvsastro.tiff">here</a></span><span class="c5">. </span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">PCA plot of microglia and astrocyte samples, showing just the first two principal components. It is clear from this plot that the largest differences are between the cell types. </span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span>A high resolution tiff of the 2D plot can be found </span><span class="c3"><a class="c8" href="pdfs/PCA.2D_microvsastro.tiff">here</a></span><span class="c5">.</span></p><p class="c1 c9"><span class="c5"></span></p><p class="c1"><span class="c5">We can do a few different things to compare the astrocyte and microglia results. Given the confounding between the two experiments, it is not advisable to make any direct comparisons between microglia and astrocyte samples. But we can still make comparisons.</span></p><p class="c1"><span>The simplest and most naive thing to do would be to compare the genes that are differentially expressed in similar comparisons. For example, the set of genes that are different between LPS and control for the two different cell types. We can use </span><span class="c3"><a class="c8" href="venn_comps/venn_comp.html">Venn diagrams</a></span><span class="c5">, as above, for this task. Note that, as with the previous Venn diagrams, we require that the genes in any intersection between two or more comparisons have to be significant in both comparisons, as well as having the same direction of change.</span></p><h2 class="c1 c21"><span class="c23 c18"></span></h2><p class="c22"><span class="c16">R packages used</span></p><p class="c1"><span class="c26">## R version 3.3.1 (2016-06-21)<br>## Platform: x86_64-pc-linux-gnu (64-bit)<br>## Running under: Debian GNU/Linux 8 (jessie)<br>## <br>## locale:<br>## &nbsp;[1] LC_CTYPE=en_US.UTF-8 &nbsp; &nbsp; &nbsp; LC_NUMERIC=C &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[3] LC_TIME=en_US.UTF-8 &nbsp; &nbsp; &nbsp; &nbsp;LC_COLLATE=en_US.UTF-8 &nbsp; &nbsp;<br>## &nbsp;[5] LC_MONETARY=en_US.UTF-8 &nbsp; &nbsp;LC_MESSAGES=en_US.UTF-8 &nbsp; <br>## &nbsp;[7] LC_PAPER=en_US.UTF-8 &nbsp; &nbsp; &nbsp; LC_NAME=C &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[9] LC_ADDRESS=C &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; LC_TELEPHONE=C &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C &nbsp; &nbsp; &nbsp; <br>## <br>## attached base packages:<br>## &nbsp;[1] grid &nbsp; &nbsp; &nbsp;stats4 &nbsp; &nbsp;parallel &nbsp;stats &nbsp; &nbsp; graphics &nbsp;grDevices utils &nbsp; &nbsp;<br>## &nbsp;[8] datasets &nbsp;methods &nbsp; base &nbsp; &nbsp; <br>## <br>## other attached packages:<br>## &nbsp;[1] latticeExtra_0.6-28 &nbsp; &nbsp;RColorBrewer_1.1-2 &nbsp; &nbsp; lattice_0.20-34 &nbsp; &nbsp; &nbsp; <br>## &nbsp;[4] KEGGREST_1.14.0 &nbsp; &nbsp; &nbsp; &nbsp;rtracklayer_1.34.2 &nbsp; &nbsp; goseq_1.26.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[7] geneLenDataBase_1.10.0 BiasedUrn_1.07 &nbsp; &nbsp; &nbsp; &nbsp; ComplexHeatmap_1.12.0 <br>## [10] readr_1.0.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;biomaRt_2.30.0 &nbsp; &nbsp; &nbsp; &nbsp; GenomicFeatures_1.26.3<br>## [13] AnnotationDbi_1.36.2 &nbsp; GenomicRanges_1.26.4 &nbsp; GenomeInfoDb_1.10.3 &nbsp; <br>## [16] IRanges_2.8.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;S4Vectors_0.12.2 &nbsp; &nbsp; &nbsp; AnnotationHub_2.6.5 &nbsp; <br>## [19] tximport_1.2.0 &nbsp; &nbsp; &nbsp; &nbsp; xtable_1.8-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; htmlwidgets_0.8 &nbsp; &nbsp; &nbsp; <br>## [22] threejs_0.2.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Rsubread_1.24.2 &nbsp; &nbsp; &nbsp; &nbsp;ReportingTools_2.14.0 <br>## [25] affycoretools_1.46.5 &nbsp; Biobase_2.34.0 &nbsp; &nbsp; &nbsp; &nbsp; BiocGenerics_0.20.0 &nbsp; <br>## [28] ggplot2_2.2.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;reshape2_1.4.2 &nbsp; &nbsp; &nbsp; &nbsp; openxlsx_4.0.0 &nbsp; &nbsp; &nbsp; &nbsp;<br>## [31] edgeR_3.16.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; limma_3.30.13 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;knitr_1.15.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [34] BiocStyle_2.2.1 &nbsp; &nbsp; &nbsp; &nbsp;rmarkdown_1.3 &nbsp; &nbsp; &nbsp; &nbsp; <br>## <br>## loaded via a namespace (and not attached):<br>## &nbsp; [1] backports_1.0.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; GOstats_2.40.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp; [3] circlize_0.3.10 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Hmisc_4.0-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp; [5] plyr_1.8.4 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;lazyeval_0.2.0.9000 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp; [7] GSEABase_1.36.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; splines_3.3.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp; [9] BiocParallel_1.8.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;digest_0.6.12 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[11] foreach_1.4.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; BiocInstaller_1.24.0 &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[13] ensembldb_1.6.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; htmltools_0.3.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[15] viridis_0.3.4 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; GO.db_3.4.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[17] gdata_2.17.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;magrittr_1.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[19] checkmate_1.8.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; memoise_1.0.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[21] BSgenome_1.42.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; cluster_2.0.6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[23] gcrma_2.46.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Biostrings_2.42.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[25] annotate_1.52.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; R.utils_2.5.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[27] ggbio_1.22.4 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;colorspace_1.3-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[29] RCurl_1.95-4.8 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;jsonlite_1.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[31] graph_1.52.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;genefilter_1.56.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[33] survival_2.41-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; VariantAnnotation_1.20.3 &nbsp; &nbsp; <br>## &nbsp;[35] iterators_1.0.8 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; gtable_0.2.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[37] zlibbioc_1.20.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; XVector_0.14.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[39] GetoptLong_0.1.6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;kernlab_0.9-25 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[41] shape_1.4.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; DEoptimR_1.0-8 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[43] prabclus_2.2-6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;scales_0.4.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[45] mvtnorm_1.0-6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; DBI_0.6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[47] GGally_1.3.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Rcpp_0.12.10 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[49] htmlTable_1.9 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; mclust_5.2.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[51] foreign_0.8-67 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;bit_1.1-12 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[53] OrganismDbi_1.16.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;preprocessCore_1.36.0 &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[55] Formula_1.2-1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; AnnotationForge_1.17.2 &nbsp; &nbsp; &nbsp; <br>## &nbsp;[57] httr_1.2.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;gplots_3.0.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[59] fpc_2.1-10 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;modeltools_0.2-21 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[61] acepack_1.4.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ff_2.2-13 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[63] flexmix_2.3-13 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;reshape_0.8.6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[65] XML_3.98-1.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;R.methodsS3_1.7.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[67] nnet_7.3-12 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; locfit_1.5-9.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[69] munsell_0.4.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; tools_3.3.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[71] RSQLite_1.1-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; evaluate_0.10 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[73] stringr_1.2.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; yaml_2.1.14 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[75] robustbase_0.92-7 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; oligoClasses_1.36.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[77] caTools_1.17.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;dendextend_1.4.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[79] nlme_3.1-131 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;RBGL_1.50.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[81] whisker_0.3-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; mime_0.5 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[83] R.oo_1.21.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; compiler_3.3.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[85] png_0.1-7 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; curl_2.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[87] interactiveDisplayBase_1.12.0 affyio_1.44.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[89] PFAM.db_3.4.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; tibble_1.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[91] geneplotter_1.52.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;stringi_1.1.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[93] trimcluster_0.1-2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Matrix_1.2-8 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[95] GlobalOptions_0.0.11 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;data.table_1.10.4 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## &nbsp;[97] bitops_1.0-6 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;httpuv_1.3.3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## &nbsp;[99] R6_2.2.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;affy_1.52.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [101] hwriter_1.3.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; KernSmooth_2.23-15 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## [103] gridExtra_2.2.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; codetools_0.2-15 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## [105] dichromat_2.0-0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; MASS_7.3-45 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [107] gtools_3.5.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;assertthat_0.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## [109] SummarizedExperiment_1.4.0 &nbsp; &nbsp;DESeq2_1.14.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [111] Category_2.40.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; rprojroot_1.2 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [113] rjson_0.2.15 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;GenomicAlignments_1.10.1 &nbsp; &nbsp; <br>## [115] Rsamtools_1.26.1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;mgcv_1.8-17 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [117] diptest_0.75-7 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;rpart_4.1-10 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <br>## [119] class_7.3-14 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;biovizBase_1.22.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;<br>## [121] shiny_1.0.0 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; base64enc_0.1-3</span></p><h1 class="c1 c20"><span class="c11"></span></h1><p class="c6"><span class="c18">References</span></p><ol class="c25 lst-kix_hz3552dmzdc3-0 start" start="1"><li class="c1 c17"><span>Smyth, G.K. 2004. &ldquo;Linear Models and Empirical Bayes Methods for Assessing Differential Expression in Microarray Experiments.&rdquo; </span><span class="c19">Statistical Applications in Genetics and Molecular Biology</span><span class="c5">&nbsp;3: Article 3.</span></li></li></ol></body></html>
